microsoft/PowerToys
# [FancyZones] Fix for the scenario of layout reset when opening the FZEditor

## Summary of the Pull Request

* Updated virtual desktops synchronization
* Updated editor launch data
* Save info about the last used virtual desktop ID
* Added unit tests


## PR Checklist



## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

* Create, delete, switch virtual desktops while FZ is working or not. Verify correct layouts are applied. 
* Clean up virtual desktops in registry and restart FZ, verify last used layout is applied.
